index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio - ZC</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./icons/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <!-- Toolbar -->
+    <div class="toolbar">
+        <p id="toolbartext">Welcome</p>
+        <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
```

Comment:
- [ ] Kindly 😊 try to put The hamburger button at the ``` nav``` tag The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.💯

> 😊⤵️

| hamburger button |
| --- |
| ![image](https://user-images.githubusercontent.com/83360636/219173848-a4f147fb-ed7c-4314-8b5c-d33f39f02996.png)  |